jobs/uaa-customized/spec

```diff
@@ -9,7 +9,32 @@ templates:
   bin/health_check: bin/health_check
 
   resources/oss/stylesheets/application.css: resources/oss/stylesheets/application.css
+  resources/oss/stylesheets/govuk-frontend-2.11.0.min.css: resources/oss/stylesheets/govuk-frontend-2.11.0.min.css
   resources/oss/images/product-logo.png: resources/oss/images/product-logo.png
   resources/oss/images/square-logo.png: resources/oss/images/square-logo.png
+  resources/oss/images/icon-pointer.png: resources/oss/images/icon-pointer.png
+  resources/oss/images/icon-pointer-2x.png: resources/oss/images/icon-pointer-2x.png
+  resources/oss/images/govuk-crest.png: resources/oss/images/govuk-crest.png
+  resources/oss/images/govuk-crest-2x.png: resources/oss/images/govuk-crest-2x.png
+  resources/oss/fonts/bold-a2452cb66f-v1.woff2: resources/oss/fonts/bold-a2452cb66f-v1.woff2
+  resources/oss/fonts/bold-f38c792ac2-v1.woff: resources/oss/fonts/bold-f38c792ac2-v1.woff
+  resources/oss/fonts/bold-fb2676462a-v1.eot: resources/oss/fonts/bold-fb2676462a-v1.eot
+  resources/oss/fonts/bold-tabular-357fdfbcc3-v1.eot: resources/oss/fonts/bold-tabular-357fdfbcc3-v1.eot
+  resources/oss/fonts/bold-tabular-784c21afb8-v1.woff: resources/oss/fonts/bold-tabular-784c21afb8-v1.woff
+  resources/oss/fonts/bold-tabular-b89238d840-v1.woff2: resources/oss/fonts/bold-tabular-b89238d840-v1.woff2
+  resources/oss/fonts/light-2c037cf7e1-v1.eot: resources/oss/fonts/light-2c037cf7e1-v1.eot
+  resources/oss/fonts/light-458f8ea81c-v1.woff: resources/oss/fonts/light-458f8ea81c-v1.woff
+  resources/oss/fonts/light-f38ad40456-v1.woff2: resources/oss/fonts/light-f38ad40456-v1.woff2
+  resources/oss/fonts/light-tabular-498ea8ffe2-v1.eot: resources/oss/fonts/light-tabular-498ea8ffe2-v1.eot
+  resources/oss/fonts/light-tabular-62cc6f0a28-v1.woff: resources/oss/fonts/light-tabular-62cc6f0a28-v1.woff
+  resources/oss/fonts/light-tabular-851b10ccdd-v1.woff2: resources/oss/fonts/light-tabular-851b10ccdd-v1.woff2
 
-properties: {}
+
+  web/layouts/main.html: web/layouts/main.html
+  web/login.html: web/login.html
+  web/forgot_password.html: web/forgot_password.html
+
+properties:
+  region:
+    description: "AWS Region Name"
+    default: "London"
```

Comment:
I wouldn't make this property have a default, we're always going to pass in a value and it should error if this is omitted